src/main/java/io/axoniq/axonserver/connector/admin/AdminChannel.java

```diff
@@ -73,4 +90,156 @@
+     */
+    CompletableFuture<List<ContextOverview>> getAllContexts();
+
+    /**
+     * Subscribes and listens to context updates from Axon Server, like when a context is created or deleted.
+     *
+     * @return a {@link ResultStream} streams context updates.
+     */
+    ResultStream<ContextUpdate> subscribeContextUpdates();
+
+    /**
+     * Request to create replication group.
+     * Returns a {@link CompletableFuture} that completes when the request has been processed by AxonServer.
+     *
+     * @param request the request to create replication group,
+     * containing the replication group name, the node name, role...
+     * @return a {@link CompletableFuture} that completes when the request has been processed to AxonServer
+     */
+    CompletableFuture<Void> createReplicationGroup(CreateReplicationGroupRequest request);
+
+    /**
+     * Request to delete replication group in Axon Server.
+     *
+     * @param request the request to delete the replication group,
+     * containing the replication group name and option to preserve event store.
+     * @return a {@link CompletableFuture} that completes when context has been deleted
+     */
+    CompletableFuture<Void> deleteReplicationGroup(DeleteReplicationGroupRequest request);
+
+    /**
+     * Request to retrieve replication group overview in Axon Server.
+     *
+     * @param replicationGroup is the name of the replication group to retrieve
+     * @return a {@link CompletableFuture} containing replication group data such as members, contexts roles, etc.
+     */
+    CompletableFuture<ReplicationGroupOverview> getReplicationGroup(String replicationGroup);
+
+    /**
+     * Request to list all replication groups in Axon Server.
+     *
+     * @return a {@link CompletableFuture} that contains list of all replication group in Axon Server.
+     */
+    CompletableFuture<List<ReplicationGroupOverview>> getAllReplicationGroups();
+
+    /**
+     * Request to add a node to replication group.
+     * Returns a {@link CompletableFuture} that completes when the request has been processed by AxonServer.
+     *
+     * @param request the request to add a node to replication group,
+     * containing the replication group name, the node name, role...
+     * @return a {@link CompletableFuture} that completes when the request has been processed to AxonServer
+     */
+    CompletableFuture<Void> addNodeToReplicationGroup(JoinReplicationGroup request);
+
+    /**
+     * Request to remove a node from replication group.
+     * Returns a {@link CompletableFuture} that completes when the request has been processed by AxonServer.
+     *
+     * @param request the request to add a node to replication group,
+     * containing the replication group name and option to preserve event store.
+     * @return a {@link CompletableFuture} that completes when the request has been processed to AxonServer
```

Comment:
I'm suggesting this change as we should be consistent in either calling it "AxonServer" or "Axon Server."
```suggestion
     * Returns a {@link CompletableFuture} that completes when the request has been processed by Axon Server.
     *
     * @param request the request to add a node to replication group,
     * containing the replication group name and option to preserve event store.
     * @return a {@link CompletableFuture} that completes when the request has been processed to Axon Server
```